src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  die() {
+    Animal.alive = Animal.alive.filter((animal) => animal !== this);
+  }
```

Comment:
The `die` method correctly removes the animal from the `Animal.alive` array using the `filter` method. However, consider checking if the `Animal.alive` array actually contains the animal before attempting to filter it out. This would prevent unnecessary filtering when the animal is not present in the array, although this is a rare edge case.